kubesae/utils.py

```diff
@@ -145,7 +145,42 @@ def count_backups(c, bucket_identifier='caktus-hosting-services-backups', profil
         print(f"{v:03d}: {k}\n")
 
 
+@invoke.task
+def scale_app(c, down=False, celery=False, container_count=2, celery_count=1):
+    """ A utility to simplify scaling namespace app pods and optionally celery pods.
+
+    Developed primarily to assist with backup verifications. Following the instructions in
+    https://caktus.github.io/developer-documentation/reference/backups/
+
+    If the namespace is using a standard django-k8s celery deployment, you can specify --celery in the command to
+    scale the celery worker and beat appropriately.
+
+    Default behavior is non-destructive, if you accidentally call this task it will merely be a no-op on a
+    running stack. However, if you wish to scale the app down to zero so the database can be dropped the usage is:
+
+    Usage:
+        inv staging utils.scale-app --down
+        inv staging utils.scale-app --down --celery
+
+        inv staging utils.scale-app --celery  # if you are scaling up a project with celery
+    """
+
+    if down:
+        c.run(f"kubectl scale -n {c.config.namespace} {c.config.container_name} --replicas=0")
```

Comment:
```suggestion
        c.run(f"kubectl scale -n {c.config.namespace} deploy/{c.config.container_name} --replicas=0")
```
Thanks for making the change, though it looks like this is now missing a "deploy/" just before the container name. For reference, something like this seems to work: https://github.com/caktus/invoke-kubesae/blob/main/kubesae/pod.py#L11

Reply:
:smek:

Reply:
☕